tests/kvmtest.sh

```diff
@@ -57,6 +57,7 @@ dhcp_relay/test_dhcp_relay.py \
 lldp/test_lldp.py \
 ntp/test_ntp.py \
 pc/test_po_cleanup.py \
+platform_tests/test_advanced_reboot.py \
```

Comment:
is this the right position? should we add to as the first entry?

Reply:
Moved up the advanced reboot tests. Since `pc/test_po_cleanup.py` has a `config reload` the test, I moved it to the last one to execute. 
Also, the run_tests.sh script had a logic to sort the tests according to the alphabetical order. This would mess up the order we want it to run tests. I think there is limited benefit to run tests in alphabetical order. Therefore, I updated run_tests.sh so that the tests are run in the order of command line input